cppForSwig/cryptopp/configure.ac

```diff
@@ -50,7 +52,13 @@ AC_CHECK_TYPES([ptrdiff_t])
 AC_FUNC_MALLOC
 AC_CHECK_FUNCS([gethostbyname gettimeofday memmove memset pow select setlocale socket])
 
-AM_CONDITIONAL([CRYPTOPP_HAVE_CLANG], [test $CXX == clang++])
+case $host in
+  *darwin*)
+  BUILD_OS=darwin
+  ;;
+esac
+
+AM_CONDITIONAL([CRYPTOPP_HAVE_CLANG], [test x$BUILD_OS = xdarwin])
```

Comment:
Yeah, I was going to clean that up later. I can fix it now.

Regarding GCC and OS X, I think you can build but a vast majority use Xcode, which defines GCC as a front for Clang (i.e., gcc = clang, g++ = clang++, etc.).